docs/ingest_data.md

```diff
@@ -0,0 +1,12 @@
+# Ingest data samples to Substra
+
+Data samples should be ingested to Substra using our python SDK or our command line
```

Comment:
Are there any other ways than cli/sdk? If not then we should use "can" instead of "should".

Reply:
If there are other ways, I don't think we want to document them.
I will update "should" to "can".